### DISCUSSION TITLE
Pls help solving the attached issue

### DISCUSSION THREAD

**Discussion by buptzouy** [milvus-io/milvus#17092] [Q&A and General discussion]
milvus-querynode exited abnormally and cannot restart succeed

<img width="1340" alt="image" src="https://user-images.githubusercontent.com/3134919/169034598-676d7fb3-90f8-4313-a78e-d3cd951fd7f4.png">

**Comment by yah01** [comment]
hi @buptzouy 
What's the version of milvus?
It would help a lot if you could provide the full log file

**Reply by buptzouy** [reply]
> hi @buptzouy What's the version of milvus? It would help a lot if you could provide the full log file

I fixed it by upgrading from. v2.0.1. to v2.0.2, thanks
